src/compas_view2/objects/vectorobject.py

```diff
@@ -0,0 +1,42 @@
+from .object import Object
+from ..buffers import make_index_buffer, make_vertex_buffer
+
+
+class VectorObject(Object):
+    """Object for displaying vector as arrow sprite."""
```

Comment:
remove "sprite". the user doesn't need to know the details :)

Reply:
Sure gonna also annotate everything properly once all changes are done